src/printer/RealTimeOIOutputBuilder.cc

```diff
@@ -0,0 +1,44 @@
+#include "RealTimeOIOutputBuilder.h"
+
+#include <sstream>
+
+namespace s2j {
+namespace printer {
+
+const std::string RealTimeOIOutputBuilder::FORMAT_NAME = "oireal";
+
+std::string RealTimeOIOutputBuilder::dump() const {
+    KillReason reason = killReason_;
+    if (reason == KillReason::NONE) {
+        if (killSignal_ > 0 || exitStatus_ > 0) {
+            reason = KillReason::RE;
+        }
+    }
+
+    std::stringstream ss;
+    ss << killReasonName(reason) << " " << exitStatus_ << " "
+       << milliSecondsElapsed_ << " " << realMilliSecondsElapsed_ << " " 
```

Comment:
I see you've added a new field here, but maybe putting realMilliSecondsElapsed in place of either miliSecondsElapsed or that 0ULL below, it'd be easier to adapt sioworkers and oiejq to parse this?

Reply:
You are right, I will replace milliSecondsElapsed